docs/lib/restapi/fragments/android/getting-started/30_initapi.md

```diff
@@ -19,8 +18,7 @@ public class MyAmplifyApp extends Application {
         super.onCreate();
 
         try {
-            // Add these lines to add the AWSCognitoAuthPlugin and AWSApiPlugin plugins
-            Amplify.addPlugin(new AWSCognitoAuthPlugin());
+            // Add this line to add the AWSApiPlugin plugin
```

Comment:
at some point a developer will have to deal with `amplify update api` and restrict access. currently this is more in line with API graphQL docs which use API Key. but i wonder, if we should consolidate ios REST API to not use auth as well, and have a new section that shows them how to `amplify update api`, restrict access, and add auth plugin

Reply:
Not opposed.